docs/news.d/idl-map.rst

```diff
@@ -1,5 +1,25 @@
 .. news-prs: 4999
 
 .. news-start-section: Additions
-- IDL Maps are now supported by opendds_idl
+.. news-rank: 10
+- IDL Maps are now supported by ``opendds_idl``.
+
+  - We would like to thank tmayoff for contributing much of the work needed for this.
```

Comment:
Maybe include a link to tmayoff's personal site, for example his GitHub account?